src/components/MembersSectionNew/MembersSectionMain/MembersSectionMainPresentation.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box } from '@chakra-ui/react';
+import { MemberProps } from '../types/MembersSection.type';
+import MembersCard from '../components/MembersCard/MembersCard';
+import MembersSectionSkeleton from './MembersSectionSkeleton';
+import styles from './MembersSection.module.css';
+
+export default function MembersSectionMainPresentation({
+  data,
+  isLoading,
+}: MemberProps) {
+  if (isLoading) return <MembersSectionSkeleton />;
```

Comment:
this can be also written in this way

```
let memberData;

if (isLoading) {
	memberData = <MembersSectionSkeleton />
} else {
	memberData = data.map((member) => (<MembersCard member={member} key={member.id} />))
}
      
      return(<>{memberData}</>)
```

which makes it more readable and easier to add cases or handle anything later

Reply:
done